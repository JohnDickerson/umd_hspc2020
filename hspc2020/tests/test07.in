75

10 16 3 19 5 3 11 4 12 0 24 
10 19 13 18 0 14 21 6 16 21 1 
10 3 24 4 9 17 14 7 10 5 6 
10 2 25 0 18 7 12 4 19 11 1 
10 17 14 2 8 3 12 16 23 12 21 
10 2 8 14 17 6 16 21 6 4 4 
10 20 20 19 10 22 23 16 15 16 0 
10 18 10 10 8 0 4 6 25 9 24 
10 15 20 13 23 7 20 21 13 6 6 
10 0 10 25 6 25 0 23 20 3 11 
10 13 15 6 0 18 14 23 6 20 4 
10 12 23 3 24 5 25 7 11 21 16 
10 14 15 23 24 11 20 14 11 20 1 
10 13 24 17 5 25 22 5 13 5 6 
10 3 12 2 16 22 7 8 21 24 4 
10 10 2 12 21 11 6 18 24 2 7 
10 2 12 8 16 10 19 24 22 11 20 
10 11 11 22 8 4 20 21 5 4 25 
10 2 10 21 14 9 13 15 15 14 5 
10 22 23 5 1 20 11 1 18 12 18 
10 7 6 3 7 10 15 7 5 14 10 
10 21 5 14 23 19 21 12 8 21 10 
10 5 25 24 25 4 7 13 13 10 24 
10 6 7 22 12 17 21 19 21 18 24 
10 1 23 14 10 21 6 6 2 4 21 
100 3 3 0 0 0 2 3 0 0 0 1 0 1 1 3 0 0 1 0 2 3 2 3 0 1 1 2 1 0 0 1 2 1 0 3 0 1 0 3 2 1 3 2 1 1 3 3 1 1 2 1 0 0 0 3 1 2 2 3 2 0 0 1 0 1 3 1 2 3 0 1 1 0 2 3 0 2 3 0 0 0 2 3 1 3 1 3 2 0 1 2 0 0 2 2 2 0 3 3 0 
100 2 3 2 3 2 0 0 2 0 1 0 0 3 0 0 2 1 1 3 0 2 2 0 3 3 3 3 1 0 3 2 3 0 1 3 0 1 0 0 2 2 2 3 1 3 1 2 2 2 3 0 1 2 0 3 0 2 0 3 3 2 2 0 0 0 0 1 3 3 2 3 2 3 2 3 1 0 3 0 0 0 0 0 0 3 3 3 3 0 0 2 2 3 2 0 2 2 0 2 1 
100 2 0 2 3 0 1 0 2 2 0 0 0 1 0 3 1 1 3 1 0 1 0 0 2 3 2 3 1 2 0 2 0 1 1 3 1 3 1 3 3 1 1 2 2 0 3 3 3 1 3 2 2 2 2 0 2 1 2 0 1 2 1 2 2 1 2 3 2 0 3 2 1 3 2 1 3 0 3 3 3 0 1 0 1 1 2 3 0 1 3 0 3 1 3 3 3 2 0 3 1 
100 2 1 1 3 3 2 0 0 3 1 3 0 0 0 2 0 1 2 1 0 2 3 1 3 2 2 2 2 0 0 1 1 1 3 2 3 2 3 1 3 2 0 0 0 2 2 2 1 2 1 3 3 3 3 3 3 1 1 0 3 2 1 0 3 2 1 2 3 2 3 3 1 1 1 0 3 3 2 3 0 0 1 2 0 3 1 2 3 0 2 3 2 0 3 0 3 0 0 3 1 
100 1 3 0 2 1 3 2 2 2 0 1 2 3 3 1 2 3 2 1 3 0 1 2 0 0 3 1 0 0 3 3 1 3 1 3 0 1 2 2 2 2 3 1 0 1 0 3 3 1 0 0 1 2 3 3 3 0 2 3 0 1 0 3 0 2 1 0 1 0 0 1 2 1 2 1 0 2 3 0 3 3 3 3 1 2 0 0 3 1 2 1 1 1 3 2 2 3 1 0 3 
100 0 1 3 2 0 0 2 1 1 3 2 2 3 2 3 2 2 2 1 0 3 1 2 3 3 3 1 2 2 2 0 3 3 1 3 1 1 2 1 2 3 3 2 2 2 2 1 0 3 3 3 2 1 1 3 2 2 1 3 1 0 0 1 0 1 1 1 3 0 0 3 2 2 1 0 0 0 2 0 3 2 2 1 2 2 0 0 1 2 0 0 1 2 1 3 2 2 3 2 3 
100 0 2 1 0 1 3 3 1 1 0 3 0 2 2 1 1 1 1 1 0 3 0 0 1 0 0 0 2 3 0 2 2 1 2 3 0 1 0 2 0 3 2 1 1 0 0 1 1 2 3 0 2 1 1 0 0 3 3 1 3 1 0 2 0 1 0 1 0 2 3 1 0 2 3 1 0 2 2 0 2 0 3 3 2 3 0 0 2 3 1 0 0 0 3 0 3 0 2 2 1 
100 1 3 3 3 2 0 2 0 3 2 1 2 1 3 0 1 2 3 2 0 0 3 2 0 0 2 3 0 0 1 1 1 2 2 3 1 3 0 3 1 2 1 0 0 2 0 3 2 1 1 2 2 1 1 2 0 0 1 0 1 1 0 3 0 3 2 1 3 0 0 1 0 1 2 0 2 0 2 1 0 3 0 1 3 1 2 1 3 3 1 0 1 2 0 3 0 1 0 0 1 
100 0 3 0 1 3 0 2 1 2 2 3 0 3 2 0 2 0 0 2 3 2 1 2 3 2 0 1 2 3 2 1 2 3 1 3 1 2 0 3 0 3 3 1 0 1 0 3 1 2 3 1 2 0 2 0 2 1 1 0 2 0 3 2 1 0 3 1 2 0 2 2 0 0 0 0 2 3 2 3 0 0 1 0 1 1 1 2 2 3 0 3 1 1 2 0 1 0 1 3 1 
100 3 2 3 2 3 1 0 0 2 0 3 3 3 2 2 3 3 1 1 3 1 3 2 1 0 1 0 0 2 3 0 0 1 2 2 2 2 1 3 2 2 2 2 0 0 0 1 1 0 3 2 0 2 2 2 3 1 2 1 0 0 2 0 2 1 3 1 2 1 2 2 3 1 0 1 1 1 2 3 2 3 3 2 0 3 2 2 2 2 3 2 0 1 3 1 3 3 0 3 1 
100 1 1 1 1 1 0 1 0 3 1 2 2 2 3 3 3 3 2 2 2 0 2 1 3 1 2 3 1 2 0 2 2 2 1 0 2 2 2 1 2 1 2 3 2 2 1 3 3 3 3 3 3 1 0 2 1 1 1 0 1 0 0 0 1 1 2 3 3 0 3 3 2 2 2 2 2 1 1 1 2 3 0 2 2 3 1 0 1 1 3 0 0 3 0 1 0 2 2 3 0 
100 1 1 1 1 2 3 3 2 1 0 2 1 3 3 1 0 3 1 3 0 0 0 3 0 3 3 0 1 3 0 0 2 3 1 1 2 1 3 0 0 0 2 1 3 2 1 1 1 2 2 1 1 2 2 0 1 1 0 0 1 3 1 1 1 3 3 3 0 0 0 3 2 2 3 0 2 1 1 0 3 0 3 0 2 3 1 0 2 2 0 1 2 3 1 0 0 0 1 2 0 
100 2 1 2 2 0 0 0 1 1 1 0 0 0 3 0 0 0 1 0 3 2 2 2 2 3 3 1 3 3 1 1 2 3 0 1 1 1 1 0 3 2 1 3 0 3 2 0 2 0 1 2 2 2 1 1 2 1 0 3 1 0 2 3 1 0 1 0 1 2 1 0 1 2 0 1 2 0 0 1 1 0 2 1 3 1 3 1 3 1 2 1 2 1 2 2 1 1 0 2 2 
100 0 2 0 1 0 0 2 0 2 3 0 0 0 2 0 0 1 3 2 0 0 1 0 3 1 2 2 0 2 2 2 2 3 0 3 3 1 3 2 1 0 0 2 3 3 2 1 2 2 1 1 1 1 3 2 2 0 3 3 3 1 0 3 3 2 2 0 0 3 0 1 0 2 1 1 3 0 1 1 2 0 0 1 1 1 3 2 3 3 0 0 1 3 1 2 0 3 1 3 1 
100 1 1 1 2 3 1 2 2 3 0 2 0 2 2 1 0 1 3 0 2 0 2 2 3 2 2 1 1 2 3 3 0 3 2 2 2 3 1 2 1 2 3 1 0 3 2 2 1 0 1 1 2 3 1 2 0 3 0 1 3 3 3 1 3 3 1 2 1 2 2 2 1 3 1 3 2 0 2 0 2 1 3 2 3 1 1 1 2 3 0 3 2 3 2 2 0 1 0 1 3 
100 2 3 2 2 0 0 3 3 0 2 3 3 2 0 3 0 1 0 3 1 3 0 2 0 1 2 1 3 2 3 3 0 1 1 0 1 3 3 2 2 2 1 2 0 3 2 0 1 0 2 2 2 3 0 0 0 0 0 1 2 0 1 0 0 3 2 0 3 2 1 1 1 0 2 2 3 0 3 1 2 1 0 1 1 1 3 0 0 0 0 3 1 0 3 3 3 2 2 0 3 
100 2 1 1 3 3 0 0 2 1 1 1 2 3 1 1 0 1 2 3 3 0 0 2 3 2 3 3 2 2 1 1 0 3 0 2 1 1 1 2 0 0 3 2 2 2 1 3 2 0 0 2 3 3 0 3 2 3 0 0 2 1 2 3 3 3 0 3 1 0 1 2 3 1 1 2 2 0 2 2 3 1 3 0 3 3 0 1 3 1 2 3 3 3 1 1 2 1 3 1 3 
100 0 1 3 0 0 3 3 1 1 1 2 0 3 3 3 0 0 0 1 1 2 0 3 1 2 3 1 3 1 1 3 0 1 3 0 2 0 1 2 0 1 0 1 2 3 0 3 0 2 2 1 0 2 3 3 2 0 2 0 1 1 0 0 3 3 3 0 2 2 1 1 2 1 3 2 0 0 1 2 3 0 0 2 2 1 3 1 0 2 3 2 0 1 1 0 0 2 1 2 0 
100 0 0 2 0 0 2 3 0 0 1 1 1 3 2 3 2 3 0 3 2 2 0 1 3 2 2 2 3 2 3 0 1 2 0 0 2 2 1 1 0 1 3 1 1 3 0 3 1 1 1 1 3 0 3 2 2 3 0 1 3 0 2 3 1 1 1 2 2 0 2 2 2 1 3 1 2 3 2 0 0 1 0 2 1 1 1 2 2 0 2 3 0 0 2 3 3 0 2 0 2 
100 2 0 0 3 0 2 3 0 2 1 2 1 1 0 2 1 2 2 0 1 3 3 3 2 0 1 0 1 3 3 0 0 0 2 1 0 0 2 3 2 0 2 3 0 0 1 2 1 0 3 1 1 2 0 0 2 0 1 2 2 1 3 1 1 3 3 1 2 2 0 1 0 0 1 0 2 3 0 1 1 0 0 2 1 1 0 3 0 2 3 0 0 2 1 1 0 3 3 3 3 
100 2 1 2 1 3 3 0 3 2 1 2 1 2 2 3 2 2 0 0 0 1 1 3 2 2 0 2 3 1 1 1 2 3 3 3 2 3 2 3 3 0 3 2 2 2 3 3 0 3 0 2 1 1 3 3 1 2 0 0 0 3 1 2 2 1 3 2 2 2 2 2 3 3 2 1 2 1 1 2 1 1 3 0 1 3 1 2 0 3 1 0 2 1 0 3 3 2 0 3 3 
100 3 0 0 1 0 0 2 1 0 1 0 3 0 3 2 1 2 0 1 3 1 0 3 1 2 2 3 3 1 2 0 2 3 0 3 2 1 3 1 3 2 2 0 0 0 2 3 1 3 0 3 2 2 0 2 1 1 2 0 2 3 0 1 0 1 2 1 0 3 3 2 3 2 2 2 0 2 1 1 1 2 0 0 3 3 1 1 2 1 3 2 0 1 1 0 3 0 2 1 3 
100 2 3 3 0 2 2 2 3 0 1 3 1 1 1 1 2 2 1 0 0 3 0 1 0 0 1 0 0 0 0 0 0 3 0 3 0 1 0 2 1 2 0 0 2 1 2 2 1 3 0 1 0 1 2 0 0 3 2 2 3 2 0 0 3 0 0 0 3 2 0 3 3 3 3 0 0 0 3 3 2 3 1 0 1 1 0 0 0 1 3 3 3 1 2 2 0 2 3 2 1 
100 1 0 0 0 1 1 1 0 0 1 1 0 1 2 0 0 1 1 3 1 2 2 1 2 0 1 3 3 2 3 1 2 2 1 2 2 2 3 3 3 0 2 2 0 1 0 1 3 0 2 0 0 3 2 2 3 2 0 3 0 1 3 3 2 2 0 3 3 0 0 2 3 3 2 3 2 0 1 0 3 0 3 3 1 1 0 3 0 3 0 3 3 0 3 0 1 2 2 0 2 
100 2 3 3 2 1 2 1 1 0 1 0 1 3 1 0 2 0 2 1 1 0 0 2 0 1 2 3 1 3 0 3 2 0 3 1 1 3 0 3 3 2 1 0 0 1 1 2 3 3 3 2 0 2 0 1 1 3 3 1 2 1 3 0 1 3 2 2 1 0 0 3 0 0 2 2 2 2 0 1 2 2 0 0 0 1 2 0 3 0 3 2 0 0 1 3 3 2 2 3 3 
100 5 2 0 3 2 0 3 5 2 4 1 4 5 0 1 2 5 2 4 3 3 0 2 4 4 2 3 4 3 2 4 3 0 5 0 1 1 0 1 3 4 5 5 3 2 0 1 4 0 0 2 4 3 1 4 5 2 5 1 2 3 4 1 2 5 3 1 4 5 3 1 5 5 0 4 3 3 5 0 2 3 0 0 2 5 2 5 1 5 5 3 2 3 1 5 5 5 2 0 0 
100 3 0 5 2 2 0 0 1 1 2 2 5 0 0 1 4 4 4 4 0 2 0 2 2 1 3 4 3 0 2 4 2 4 5 5 3 0 1 3 4 5 2 4 4 2 2 1 0 5 3 1 1 4 4 4 5 1 2 2 4 2 0 3 2 2 2 0 0 3 1 1 1 5 1 5 1 0 5 1 4 1 5 4 3 5 1 5 3 4 2 3 1 1 1 5 2 1 5 1 0 
100 2 4 0 2 3 4 0 1 1 1 2 3 2 4 4 3 1 5 0 4 0 3 0 1 5 5 4 4 2 0 2 3 5 2 3 3 0 3 3 3 0 2 2 4 1 1 4 5 4 0 2 3 1 1 0 0 0 4 4 1 1 1 3 1 2 4 1 2 0 0 4 5 2 2 3 1 4 3 4 0 1 4 1 5 3 0 0 0 4 3 1 4 0 0 1 4 5 2 5 1 
100 4 4 5 5 4 4 5 0 0 5 5 3 0 3 0 0 2 5 1 5 5 5 4 5 3 2 0 3 0 5 1 1 1 4 1 0 0 1 5 1 4 3 2 3 3 5 1 0 0 0 0 4 1 3 3 1 1 1 3 0 1 3 0 5 4 0 2 5 2 1 5 5 0 1 1 0 5 3 5 3 3 5 3 4 5 4 4 0 4 2 4 5 1 1 1 4 5 4 1 0 
100 4 0 5 3 0 2 2 1 5 1 5 0 0 4 4 2 3 0 5 1 0 1 0 0 1 2 4 4 5 2 2 4 5 1 2 4 3 4 5 5 4 2 5 4 1 4 5 3 0 3 0 3 1 1 4 5 3 4 4 1 1 2 5 3 3 5 5 3 2 4 2 2 4 0 2 1 4 4 2 3 5 0 3 1 1 0 5 4 1 0 4 5 3 3 3 2 2 4 3 5 
100 3 0 1 4 0 4 1 1 2 1 1 3 5 2 2 5 3 3 1 0 2 5 4 1 5 2 3 0 5 3 1 1 3 5 0 4 1 2 0 5 0 1 1 5 1 3 1 0 2 1 5 1 2 3 3 2 1 3 3 5 1 4 1 2 3 2 3 3 2 2 0 4 2 0 5 0 2 5 2 3 4 2 5 3 2 1 4 1 0 0 2 5 2 4 4 0 1 3 4 2 
100 2 5 5 2 4 2 1 4 5 1 3 2 5 5 2 2 3 4 1 4 1 1 0 4 1 0 1 2 3 2 5 0 4 2 5 2 1 2 3 1 1 4 0 0 2 5 0 2 5 1 3 4 2 2 3 2 0 1 4 5 3 3 5 5 4 3 2 3 3 1 4 0 2 4 1 2 2 0 0 5 4 1 3 3 3 1 0 3 2 1 1 5 3 5 4 4 3 5 1 3 
100 4 5 2 0 3 5 1 1 3 2 2 2 4 4 3 4 0 4 4 3 1 5 4 1 2 5 3 3 2 5 0 3 1 2 1 5 3 0 1 2 1 0 4 4 3 3 2 1 1 4 1 3 5 3 2 0 5 1 2 5 2 3 4 4 3 4 5 4 2 5 1 0 1 0 2 5 0 5 4 0 4 4 2 0 5 0 0 0 4 1 5 0 1 3 4 2 1 2 5 4 
100 0 3 4 0 2 4 4 2 4 5 0 4 2 5 3 0 3 3 2 4 4 5 4 0 0 5 5 3 4 0 4 4 2 0 1 3 3 3 1 5 3 3 2 4 5 0 0 3 0 4 4 4 3 2 0 2 0 3 2 5 0 2 1 3 5 3 4 2 2 2 2 0 4 2 5 5 5 3 5 0 4 3 4 3 3 0 5 3 0 4 1 2 3 4 1 0 4 1 3 2 
100 1 2 5 3 1 2 5 3 4 1 4 5 2 1 3 1 4 5 4 5 2 2 4 4 0 0 5 1 0 4 3 3 2 0 0 4 4 5 3 3 1 5 4 2 1 1 0 2 3 3 5 4 0 0 5 5 3 2 5 0 4 5 2 0 5 2 5 3 5 0 5 0 3 5 0 4 5 2 4 2 5 0 4 1 4 5 4 2 3 5 1 4 0 1 3 1 0 4 1 4 
100 0 4 3 1 1 4 2 4 3 4 0 0 2 0 2 4 1 4 2 1 5 3 0 4 5 0 3 5 4 5 2 4 5 1 0 2 0 0 2 4 5 5 4 2 2 5 1 2 5 5 2 4 2 0 4 3 4 1 2 0 5 2 0 5 0 0 2 0 1 2 3 1 3 1 5 4 3 1 5 3 2 5 3 3 2 2 4 2 1 4 2 1 5 1 0 1 1 0 0 1 
100 0 3 1 5 1 0 5 4 3 4 0 2 1 4 1 3 1 3 0 4 4 4 1 0 1 2 3 1 0 0 0 2 3 0 3 4 0 1 0 0 4 2 1 5 4 1 5 2 2 4 4 1 1 1 3 5 3 0 3 3 3 4 5 1 0 3 4 3 1 1 2 0 4 0 4 2 2 0 0 4 5 3 5 5 5 0 5 3 2 1 2 1 5 4 1 3 5 5 4 5 
100 5 0 0 4 2 1 3 1 4 0 1 3 1 5 0 2 2 4 2 1 3 0 3 4 1 3 0 3 3 2 0 2 1 2 4 2 2 2 0 4 0 3 3 3 1 3 4 2 5 0 2 5 2 0 4 0 3 4 4 2 3 5 4 2 3 2 2 4 0 3 3 4 2 2 0 1 5 3 2 2 2 4 5 2 5 4 3 4 3 4 2 5 3 3 2 5 5 0 3 2 
100 4 3 5 3 4 1 1 1 0 1 3 5 0 0 4 1 1 5 0 2 2 3 0 2 0 0 4 2 5 0 5 5 1 0 3 1 0 0 1 2 4 0 5 4 5 0 1 1 1 4 3 1 4 5 5 3 0 3 1 5 1 3 1 5 4 4 4 4 3 2 4 1 5 4 3 5 3 1 5 1 3 2 5 5 0 3 4 4 2 4 5 2 4 1 0 2 3 0 2 0 
100 4 1 2 4 1 0 2 5 5 1 1 3 4 2 3 4 4 4 1 4 1 5 0 2 4 2 5 0 5 2 0 0 4 5 0 4 0 5 0 3 4 2 0 4 2 3 3 0 0 5 5 4 0 4 5 4 1 5 4 1 0 1 1 0 0 5 4 5 2 2 1 1 5 3 1 4 0 0 2 4 5 3 1 3 0 5 1 3 1 2 2 0 1 5 2 5 2 2 1 5 
100 5 4 0 1 5 3 0 5 3 3 0 4 2 4 0 2 1 0 5 0 1 2 3 3 1 1 1 2 3 5 0 4 2 2 5 4 3 0 0 3 1 3 5 1 0 2 4 5 3 0 5 4 1 2 2 5 3 1 1 1 2 4 4 2 0 2 3 3 0 3 5 5 2 5 0 1 4 4 0 3 3 3 3 0 2 1 4 3 5 5 2 2 2 3 3 5 4 4 2 0 
100 3 4 0 1 2 3 4 3 2 0 1 1 5 1 5 0 4 1 3 5 1 4 4 0 1 5 4 1 1 3 0 1 4 2 3 0 4 0 0 0 0 3 3 2 5 4 0 4 4 4 3 0 0 2 1 3 3 2 2 4 4 4 1 5 1 0 3 3 4 1 1 3 5 0 2 4 1 0 0 4 1 3 0 5 4 4 1 4 5 1 5 5 3 5 4 3 1 4 2 0 
100 5 4 3 3 1 2 1 1 1 2 3 2 2 2 0 0 2 5 0 2 3 0 1 4 4 0 2 0 3 2 1 3 2 1 5 4 1 2 4 2 1 3 5 2 3 1 4 1 4 2 2 2 2 4 5 4 5 0 0 5 1 2 5 1 4 2 5 5 4 2 4 4 0 5 0 5 1 0 2 0 1 1 1 0 1 1 3 2 3 1 0 3 5 3 4 0 4 4 1 2 
100 2 0 4 4 1 4 2 2 1 0 1 0 3 5 2 5 0 4 0 1 1 1 4 1 0 1 4 5 3 1 4 0 0 2 1 4 3 2 3 4 5 4 1 2 0 0 5 4 1 2 4 0 4 5 0 1 2 3 0 2 3 4 3 5 1 0 4 5 2 5 3 3 1 5 0 3 0 4 4 3 3 3 1 0 2 4 4 2 5 2 4 4 5 3 5 3 2 3 5 5 
100 3 3 3 0 4 5 4 2 4 3 5 5 0 1 1 4 5 4 1 0 5 5 5 2 0 1 2 2 3 3 4 4 0 1 4 4 2 1 0 4 2 3 2 5 4 3 2 1 2 3 2 2 5 5 4 4 0 2 2 5 3 3 1 4 3 4 4 0 1 5 4 5 5 1 0 4 0 3 2 2 0 1 1 4 1 0 3 2 1 4 1 2 3 5 5 3 4 2 0 1 
100 4 2 5 0 1 0 2 4 0 0 5 1 4 0 5 0 1 5 2 3 0 5 5 3 4 5 3 5 4 0 1 3 1 2 2 4 1 0 4 1 2 1 3 5 4 2 0 1 0 4 0 3 1 4 1 4 1 0 5 2 4 3 5 5 3 1 0 4 4 2 2 0 0 2 5 0 5 0 1 1 3 5 3 3 0 0 3 0 1 2 2 1 2 3 4 0 5 1 3 5 
100 0 5 2 5 0 5 4 2 1 5 3 5 4 3 3 2 3 0 5 4 3 4 3 0 0 3 2 0 0 3 3 2 2 2 2 0 3 3 2 0 1 4 4 1 4 2 1 4 5 2 1 5 5 1 5 1 5 0 4 0 5 0 4 5 5 3 4 2 4 3 0 3 2 1 0 2 2 0 2 1 1 1 1 3 5 4 4 2 3 2 4 3 2 3 4 3 4 1 5 5 
100 5 2 2 3 5 4 0 1 4 3 4 0 1 4 4 0 0 5 2 4 0 5 4 5 4 0 2 3 4 4 2 2 2 4 2 1 5 5 1 4 0 1 2 2 5 2 0 5 4 2 3 1 5 1 3 1 3 3 4 5 5 4 4 5 5 1 0 5 0 0 3 5 0 3 3 4 3 4 5 5 0 0 2 5 4 4 3 4 3 3 3 1 3 5 0 3 3 1 1 5 
100 5 0 4 3 0 5 2 1 5 2 1 4 5 5 3 5 2 5 2 0 3 4 3 0 3 1 2 5 3 2 0 0 4 2 4 4 2 5 4 2 3 3 4 1 0 1 1 2 3 5 2 1 4 5 1 0 1 1 0 3 0 0 0 5 0 3 2 1 0 1 5 2 3 1 0 2 5 4 2 2 1 4 1 2 4 4 2 1 4 0 5 1 4 4 4 3 4 5 3 4 
100 5 3 1 0 4 4 3 1 2 3 4 4 3 0 2 3 1 3 1 4 4 4 5 0 1 3 3 4 2 4 0 5 0 3 4 3 0 1 1 4 1 2 5 4 4 0 0 1 1 2 4 2 0 0 1 2 0 4 5 1 2 2 4 5 1 5 3 5 0 3 5 4 4 4 0 3 4 1 3 4 0 3 4 3 3 0 2 1 5 0 4 3 4 5 3 2 1 4 2 2 
