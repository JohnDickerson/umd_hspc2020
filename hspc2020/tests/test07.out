92
51
25
54
56
12
12
85
22
50
42
63
24
35
40
41
19
70
9
64
21
47
74
24
84
166
141
138
133
144
133
163
156
154
135
122
145
144
146
125
156
131
147
139
166
122
154
165
116
153
223
249
254
232
229
243
238
242
205
209
208
251
243
228
245
241
234
257
230
214
265
234
219
247
236
